pandas/core/indexes/interval.py

```diff
@@ -808,58 +814,28 @@ def get_loc(self, key, method=None):
         array([0, 1], dtype=int64)
         """
         self._check_method(method)
+        if is_list_like(key):
+            raise KeyError(key)
 
```

Comment:
can you add a comment here on what its doing

Reply:
done